### PR TITLE
feat: theme

### LINKED ISSUE #33: theme

### PR DESCRIPTION
**Pull request by Suntoh**
## What I did

- themes
- select theme components

## Note

need to be used in setting page

### CODE REVIEW

File: apps/server/.env.example
Comment (Imeany12) [inline apps/server/.env.example:1]: this file should not be deleted

File: apps/client/src/app/lobby/page.tsx
Comment (Imeany12) [inline apps/client/src/app/lobby/page.tsx:1]: A QUESTION: We don't need lobby page for now?

File: apps/client/public/images/snowBg.png
Comment (Imeany12) [inline apps/client/public/images/snowBg.png:1]: Is it ok for the watermark? 😅

File: apps/server/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "include": ["."],
   "exclude": ["build", "node_modules"],
   "compilerOptions": {
-    "outDir": "./build"
+    "outDir": "./build",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./*"]
+    }
```
Comment (Imeany12) [inline apps/server/tsconfig.json:10]: A QUESTION: is this automatically generated when you installed radix?

File: apps/client/tailwind.config.ts
Comment (Imeany12) [inline apps/client/tailwind.config.ts:1]: A QUESTION: are you still gonna you commented themes?

**Review by Imeany12 — CHANGES_REQUESTED** [review]

File: apps/client/src/app/lobby/page.tsx
Comment (Suntoh) [reply apps/client/src/app/lobby/page.tsx:1]: yes we can delete

File: apps/server/tsconfig.json (hunk 1 above)
Comment (Suntoh) [reply apps/server/tsconfig.json:10]: No I think it's because I installed cors?
I didn't touch this part of the code

File: apps/client/tailwind.config.ts
Comment (Suntoh) [reply apps/client/tailwind.config.ts:1]: I think it can be useful in the future

Comment (Suntoh) [reply apps/client/tailwind.config.ts:1]: I will uncommented it so I can use it in tailwind.

File: apps/client/public/images/snowBg.png
Comment (Suntoh) [reply apps/client/public/images/snowBg.png:1]: i can't find a better picture. for me it's kind of OK but if you want me to change I can

Comment (Suntoh) [reply apps/client/public/images/snowBg.png:1]: changed

File: apps/server/src/server.ts [hunk 2]
```diff
@@ -12,6 +13,17 @@ import userRoutes from './routes/user';
...
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader('Access-Control-Allow-Methods', 'POST, GET, PUT');
+  res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
+  next();
+});
+const corsOptions = {
+  origin: 'http://localhost:3000',
+  credentials: true,
+};
+app.use(cors(corsOptions));
```
Comment (Imeany12) [inline apps/server/src/server.ts]: I think we should allow all CORS Requests

```suggestion
app.use(cors())
```

File: apps/server/src/server.ts [hunk 3]
```diff
@@ -12,6 +13,17 @@ import userRoutes from './routes/user';
...
+app.get('/', (req, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader('Access-Control-Allow-Methods', 'POST, GET, PUT');
+  res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
+  next();
+});
+const corsOptions = {
+  origin: 'http://localhost:3000',
+  credentials: true,
+};
```
Comment (Imeany12) [inline apps/server/src/server.ts]: From below, this will not be needed

**Review by Imeany12 — CHANGES_REQUESTED** [review]

File: apps/client/src/app/lobby/[roomId]/page.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    }
+  };
+  return (
+    <div className='min-h-screen bg-gray-800 text-white'>
+      {/* Header: Player Info */}
+      <header className='flex items-center justify-between bg-gray-900 p-4'>
+        {session ? (
+          <div className='flex items-center'>
+            <Image
+              src={user?.image ?? '/images/default-profile.png'}
```
Comment (Imeany12) [inline apps/client/src/app/lobby/[roomId]/page.tsx:70]: don't forget to add fallback profile image 
image

**Review by Imeany12 — APPROVED** [review]
LGTM 👍

**Pull request merged**